Tests/SentryTests/SentryTracerTests.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+
+class SentryTracerTests: XCTestCase {
```

Comment:
`m`: Can we please add some async tests to make sure the locking in the `SentryTracer` works fine, you can take a look at `SentryConcurrentRateLimitsDictionaryTests` or `SentryScopeSwiftTests.testModifyingFromMultipleThreads`.

Reply:
I created an issue for this https://github.com/getsentry/sentry-cocoa/issues/1134